structuralalignment/superposition/theseus.py

```diff
@@ -1 +1,114 @@
-#here is the theseus code
+import atomium
+import subprocess
+
+# usage: Theseus(["3dk3"])
+class Theseus:
+    def __init__(self, pdbs):
+        self.pdbs = pdbs
+        self.fetched_pdbs = []
+        for pdb in pdbs:
+            self.fetched_pdbs.append(atomium.fetch(pdb))
+        self.align()
+
+    def align(self):
+        self.fetched_pdbs_filename = []
+        """
+        for m in range(len(self.fetched_pdbs)):
+            self.fetched_pdbs[m].model.save(self.pdbs[m])
+        """
+        for m in range(len(self.fetched_pdbs)):
+            self.fetched_pdbs[m].model.save(
+                self.pdbs[m] + "." + self.fetched_pdbs[m].filetype
+            )
+            self.fetched_pdbs_filename.append(
+                self.pdbs[m] + "." + self.fetched_pdbs[m].filetype
+            )
+
+        output = subprocess.check_output(["theseus",] + self.fetched_pdbs_filename)
```

Comment:
That trailing comma in the list is not needed

```suggestion
        output = subprocess.check_output(["theseus"] + self.fetched_pdbs_filename)
```